frontend/public/components/utils/volume-icon.jsx

```diff
@@ -3,15 +3,23 @@ import * as React from 'react';
 import {VolumeSource} from '../../module/k8s/pods';
 
 export const VolumeIcon = ({kind}) => {
+  const kindMapping = {
+    persistentVolumeClaim: 'PVC',
+  };
   const faClasses = _.fromPairs([
     [VolumeSource.emptyDir.id, 'fa-folder-open-o'],
     [VolumeSource.hostPath.id, 'fa-files-o'],
     [VolumeSource.secret.id, 'fa-lock'],
   ]);
+  const coClasses = _.fromPairs([
+    [VolumeSource.persistentVolumeClaim.id, 'co-m-resource-persistentvolumeclaim'],
+  ]);
   const faClass = faClasses[kind];
+  const coClass = coClasses[kind];
 
-  return <span className="co-icon-and-text co-m-volume-icon">
-    {faClass && <i className={`fa ${faClass} co-icon-and-text__icon`}></i>}
+  return <span className="co-icon-and-text">
+    {faClass && <i className={`fa co-m-volume-icon ${faClass} co-icon-and-text__icon`}></i>}
+    {coClass && <span className={`co-m-resource-icon ${coClass}`} title={kind}>{kindMapping[kind]}</span>}
```

Comment:
I don't think we should use the PVC resource icon here since we don't use the `S` for secret volumes or the `CM` for config map volumns.

Reply:
@julim@redhat.com

Reply:
Typically we use it for a specific resource instance with a link as well, so we're breaking the pattern.

Reply:
Which icon do you suggest we use then?

Reply:
Honestly, I'm not sure having a different icon for every kind of volume (emptydir, hostPath, secret, config map, PVC, etc.) scales. It would be difficult to find meaningful icons for all of these. We might just leave it off.

If it links to a specific secret, config map, or PVC, it would be nice to have a link to that specific object with the resource icon badge.